logbook-spring-boot-autoconfigure/src/test/java/org/zalando/logbook/autoconfigure/ExcludeTest.java

```diff
@@ -35,33 +37,33 @@
     @Autowired
     private Logbook logbook;
 
-    @MockBean(name = "httpLogger")
-    private Logger logger;
+    @MockBean
+    private HttpLogWriter writer;
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/whiskeysierra/logbook/pullRequest?prid=5950383)